src/client/containers/formComponent/formComponent.js

```diff
@@ -2,58 +2,45 @@ import React, { Component } from 'react';
 import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
-import FormField from '../../components/formField/formField';
+import RenderFormFields from '../../components/renderFormFields/renderFormFields';
 import postUserData from '../../actions/postUserDataAction';
-import getUserData from '../../actions/getUserDataAction';
 import './formComponent.css';
 
 class FormComponent extends Component {
     state = {
-      userName: '',
+      username: '',
       email: '',
       password: '',
     };
 
-
     onInputChange = ({ target: { name, value } }) => {
       this.setState({ [name]: value });
     };
 
-    onRegistratuinSubmit = (e) => {
+    onSubmit = (e) => {
       e.preventDefault();
-      const { userName, email, password } = this.state;
-      const { history } = this.props;
-      const newUser = {
-        userName,
-        email,
-        password,
-      };
-      postUserData(newUser);
-      history.push('/main');
-    };
+      const { username, email, password } = this.state;
+      const { postUserData: post, isMember, history } = this.props;
+      let path;
+      let user;
 
```

Comment:
it can be chorter let path, user; 
And if you will not reassign them, you can use const for their declaration

Reply:
But Travis doesn't think so :)